libvast/vast/table_slice_column.hpp

```diff
@@ -70,7 +71,7 @@ class table_slice_column {
   }
 
 private:
-  table_slice_ptr slice_ = nullptr;
+  table_slice slice_ = {};
```

Comment:
The explicit assignment should not be necessary.

Reply:
It is for the default constructor.